app/store/modules/product/index.js

```diff
@@ -0,0 +1,35 @@
+// store/modules/product/index.js
+import productData from './mock';
+
+const namespaced = true;
+
+// initial state
+const state = {
+  items: productData // demo
+};
+
+// getters
+const getters = {
+  /**
+   * Gets a product
+   *
+   * @param   {state}   Current state
+   * @returns  {Array} Product
```

Comment:
- [ ] Please always add a blank line before `@returns`. Should give ESLint error. Please check if running.

Reply:
Added the line but no error.